### ISSUE TITLE
[BUG] Pipelines are showing wrong input and processed values

### ISSUE THREAD

**Issue by c3s4rfred** [utmstack/UTMStack#379]
**Describe the bug**
In the Data Processing view the pipelines are showing wrong input and processed values

**To Reproduce**
1. Go to 'Data processing' right menu
2. See the differences between input and processed values

**Expected behavior**
Input and processed values must have similar (not necessary the same) values

**Screenshots**
![image](https://github.com/utmstack/UTMStack/assets/17603499/669e84c7-bd20-41d9-aabe-02188ed4d7c4)

**Environment**
 - UTMStack 10.2 RC

**Possible solution**
Check if it is a bug from logstash or backend bug

**Comment by jdieguez89** [comment]
@c3s4rfred any update?

**Comment by c3s4rfred** [comment]
Yes, isn't an UTMStack bug, seems to be a Logstash bug, I already posted the problem on Elastic Disscussion -> https://discuss.elastic.co/t/logstash-stats-api-is-showing-wrong-events-values/352419 and https://stackoverflow.com/questions/77940089/logstash-stats-api-is-showing-wrong-events-values. I'll be checking periodically if somebody answer.
I checked the configurations in the documentation and I didn't find anything about that, so I'll wait a little bit for an answer from the discussion before reporting as a bug.
